.env

```
@@ -6,6 +6,9 @@ DB_NAME = "scibblr"
 JWT_SECRET="scibblr"
 JWT_EXPIRES_IN=2d
 
+EMAIL_USER = "yd490011@gmail.com"
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)